Sources/Datadog/RUM/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -437,7 +439,7 @@ internal class RUMViewScope: RUMScope, RUMContextProvider {
                 action: .init(count: actionsCount.toInt64),
                 cpuTicksCount: cpuInfo?.greatestDiff,
                 cpuTicksPerSecond: cpuInfo?.greatestDiff?.divideIfNotZero(by: Double(timeSpent)),
-                crash: .init(count: 0),
+                crash: isCrash ? .init(count: 1) : .init(count: 0),
```

Comment:
nit.

It's annoying that Swift doesn't allow conversion from Bool to Int out of the box.

We could do something like this:
```swift
extension ExpressibleByIntegerLiteral {
    init(_ booleanLiteral: BooleanLiteralType) {
        self = booleanLiteral ? 1 : 0
    }
}

Int(isCrash)
```